### DISCUSSION TITLE
Erro de conexão com o banco

### DISCUSSION THREAD

**Discussion by RafaelFCouto** [balta-io/forum#462] [Tira Dúvidas] [answered]
Primeiro, eu fiz a criação do meu banco de dados e depois quis fazer a conexão utilizando Entity Framework e me deparo com esse erro: 
![image](https://github.com/balta-io/forum/assets/149386775/b688b923-6a29-455a-9c9a-fbe1f751e22a)

Alguém poderia ajudar?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @RafaelFCouto, como vai?
Qual curso e aula está realizando e como está sua connection string?
Consegue colocar os trechos de código onde realiza a conexão e o código onde este erro ocorreu?

Sem os detalhes não conseguimos te ajudar!

**Reply by RafaelFCouto** [reply]
Bom dia, fiz uma interação fora do escopo dos cursos da Balta.
Vi agora que não é permitido...
Desculpa o incomodo 😁

**Reply by andrebaltieri** [reply]
@RafaelFCouto relaxa, pode perguntar sim, só manda mais detalhes... fica tranquilo 🚀
